src/SmartComponents/Actions/ListActions.js

```diff
@@ -217,11 +229,18 @@ class ListActions extends Component {
 }
 
 ListActions.propTypes = {
+    breadcrumbs: PropTypes.array,
+    history: PropTypes.object,
```

Comment:
last thing then i promise we can merge this, cleaning up the props here... we dont need history path params